### ISSUE TITLE
Fix ^C handling

### ISSUE THREAD

**Issue by tych0** [anuvu/stacker#178]
Our outer userns exec wrapper doesn't forward ^C to the inner one, so things don't correctly get killed on ^C. We need to do something like what's in container.go:Execute() i guess.

**Comment by smoser** [comment]
As reported by @raharper and I, if you ctrl-c while stacker is running and has launched a container, it crashes gnu screen and *all* gnu screen processes.  Yikes.

**Comment by tych0** [comment]
That almost seems like a gnu screen bug? in tmux it just orphans the thing and dooesn't actually ^C :)

**Comment by mikemccracken** [comment]
weird. as a data point, I use gnu screen all the time and am pretty sure I've ^C'd a stacker build a few times.
never got nuked like that. I'm using bionic, with 5.4.0 though...

**Comment by tych0** [comment]
I don't think (?) this is fixed. It's "better" now, but I've still seen stacker kill tmux sessions.

**Comment by smoser** [comment]
> I don't think (?) this is fixed. It's "better" now, but I've still seen stacker kill tmux sessions.

i dont think fixed either. :-(.  its hard to remember to not hit ctrl-c if stacker is running.

**Comment by tych0** [comment]
Yeah, any sleuthing about what's going on is much appreciated. I don't really understand what's happening here.
